scripts/hyperlane/07-fantoken-test.sh

```
@@ -0,0 +1,363 @@
+  initial_balance=$("$BINARY" query bank balance "$VAL_ADDRESS" "$ft_denom" \
+    --output json --node "$NODE" --home "$BITSONG_HOME" 2>/dev/null | \
+    jq -r '.balance.amount // "0"' 2>/dev/null) || initial_balance="0"
+
+  log "Sending $transfer_amount $ft_denom: Base Sepolia -> BitSong"
+  log "  HypERC20:  $ft_evm_hyp_erc20 (supply: $evm_balance)"
+  log "  Recipient: $VAL_ADDRESS"
+  log "  Initial Cosmos balance: $initial_balance $ft_denom"
+
+  local initial_height
+  initial_height=$("$BINARY" status --node "$NODE" --home "$BITSONG_HOME" 2>/dev/null | \
+    jq -r '.sync_info.latest_block_height // "0"' 2>/dev/null) || initial_height="0"
```

Shellcheck (0.11.0)

[warning] initial_height appears unused. Verify use (or export if used externally).

(SC2034)